hazelcast-spring/src/main/java/com/hazelcast/spring/HazelcastClientBeanDefinitionParser.java

```diff
@@ -183,6 +183,8 @@ AbstractBeanDefinition createConfigBean(Node rootNode) {
                     configBuilder.addPropertyValue("instanceName", getTextContent(node));
                 } else if ("labels".equals(nodeName)) {
                     handleLabels(node);
+                } else if ("backup-ack-to-client-enabled".equals(nodeName)) {
```

Comment:
Renaming suggestion: `backup-ack-enabled`

Reply:
It is decided this way in design. In my opinion, backup ack enabled might be misunderstood. There is always backup ack, but without this, it is to the member, not to the client.